widgetastic_manageiq.py

```diff
@@ -878,3 +878,144 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class Stepper(Widget):
+    """ A CFME Stepper Control
+
+        stepper = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+        stepper.inrease()
+    """
+    def __init__(self, parent, locator, logger=None):
+        super(Stepper, self).__init__(parent=parent, logger=logger)
+
+        self._stepper_control = self.browser.element(locator, parent=parent)
+
+        self.minus_button = Button(self, '-')
+        self.plus_button = Button(self, '+')
+        self.value_field = Input(self, locator='.//input[contains(@class, '
+                                               '"bootstrap-touchspin")]')
+
+    def __locator__(self):
+        return self._stepper_control
+
+    def read(self):
+        return int(self.value_field.read())
+
+    def decrease(self):
+        self.minus_button.click()
+
+    def increase(self):
+        self.plus_button.click()
+
+    def set_value(self, value):
+        if value < 1:
+            raise ValueError('The value cannot be less than 1')
+        steps = int(value) - self.read()
+        if steps == 0:
+            return
+        elif steps > 0:
+            operation = self.increase
+        else:
+            operation = self.decrease
+
+        steps = abs(steps)
+        for step in range(steps):
+            operation()
+
+    def fill(self, value):
+        self.set_value(value)
```

Comment:
fill must return True or False - True if it changed anything, False if nothing was changed.